source/daplink/cmsis-dap/DAP_vendor.c

```diff
@@ -38,11 +38,17 @@
 #include "settings.h"
 #include "target_family.h"
 #include "flash_manager.h"
+#include "util.h"
 #include <string.h>
 #include "daplink_vendor_commands.h"
 
 #ifdef DRAG_N_DROP_SUPPORT
 #include "file_stream.h"
+#include "virtual_fs.h"
+
+// TODO: It might be more memory efficient to re-use the usb_buffer from vfs_manager.c
+static uint8_t __ALIGNED(4) file_stream_buffer[VFS_SECTOR_SIZE];
+static uint16_t file_stream_buffer_pos = 0;
```

Comment:
As described in the PR, this is the buffer that could be removed and `usb_buffer` from `vfs_manager.c` used instead.
Left it as a TODO comment to come to an agreement before merging the PR.